### DISCUSSION TITLE
Can a layer of LIF neurons have different thresholds?

### DISCUSSION THREAD

**Discussion by DS-567** [BindsNET/bindsnet#592] [General]
In BindsNet, when using LIF nodes to create a layer in a SNN, can we change each neurons firing threshold as a parameter, or do all neurons in that layer have the same threshold?

**Comment by SimonInParis** [comment]
Hi.

Yes, the threshold parameter in a neuron layer can be passed as a tensor with the same dimension as the layer.

**Reply by DS-567** [reply]
Thank you Simon, this works perfectly.
